tmt-app/src/Components/CareerPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+function CareerPage() {
+  const { state } = useLocation();
+
+  useEffect(() => {
+    console.log("testing", state.data);
+  }, [state]);
+
+  return <div>{state.data[0].name}</div>;
+}
+
+export default CareerPage;
```

Comment:
Is this a test page? Let's not commit test code

Reply:
This is the page that is routed to after clicking on the card. I was testing to see if I was able to successfully pass props to this page.

Reply:
Ah okay, let's leave the test code out, doesn't need to be committed.

You can look into url params for this. As long as you have url update accordingly, you should be okay